docroot/modules/custom/va_gov_migrate/config/install/migrate_plus.migration.office.yml

```
@@ -0,0 +1,54 @@
+uuid: a2a9db75-73ad-44ff-85fc-fe7284ec91d7
+langcode: en
+status: true
+dependencies: {  }
+id: office
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: null
+migration_group: office
+label: 'Node - Office'
+source:
+  plugin: csv
+  path: modules/custom/va_gov_migrate/data/office-salesforce.csv
+  delimiter: ','
+  enclosure: '"'
+  header_offset: 0
+  ids:
```

```suggestion
  track_changes: true
  cache_counts: true
  ids:
```

Need track changes so it only updates a node if the source row changes.

---

Got it!